### ISSUE TITLE
receiptID in tx.receipts is null ?

### ISSUE THREAD

**Issue by csquan** [eteu-technologies/near-api-go#10]
when I get receiptID in tx.receipts is null ?is a bug ?

**Comment by mikroskeem** [comment]
It is. Please provide more information on the issue next time in the form of:

1) What did you try to do (with code snippet to reproduce)
2) What did you expect to see?
3) What did you see instead?

**Comment by csquan** [comment]
I try to get receiptID from block 65906486,but it is null.
to verify it ,then I query mannet db ,find receiptID is not null.
what I expect is receiptID should be value equal to which in  mannet db.

**Comment by mikroskeem** [comment]
Please provide a code example and the query you use to find the `receiptID` from the database.

**Comment by csquan** [comment]
no code example.I just use api to get block,then get tx,receipt .
find receptID is null in every block,just test you can find it.
most likely unmarshal error.

**Comment by csquan** [comment]
![image](https://user-images.githubusercontent.com/16413009/169440439-f4d73024-6fb0-4452-a17f-93246e2d3d5a.png)

**Comment by mikroskeem** [comment]
Found the issue - https://github.com/eteu-technologies/near-api-go/blob/25bf2c59c15540ee0d38c6190c2389e36591eff5/pkg/client/transaction_structs.go#L44=

**Comment by mikroskeem** [comment]
Fixed in e8be92c43bcfabfde4e22e77cff6e549fb5e6bef

Tested with:

```go
package main

import (
	"context"

	"github.com/eteu-technologies/near-api-go/pkg/client"
	"github.com/eteu-technologies/near-api-go/pkg/config"
	"github.com/eteu-technologies/near-api-go/pkg/types/hash"

	"github.com/davecgh/go-spew/spew"
)

var (
	txnHash = hash.MustCryptoHashFromBase58("AFYNx2sJ64Vwqx5LzgFbmhJ5hJbM8xFLvn93KwAW9YaL")
	account = "app.nearcrowd.near"
)

func main() {
	c := config.Networks["archival-mainnet"]
	rpc, err := client.NewClient(c.NodeURL)
	if err != nil {
		panic(err)
	}

	ctx := context.Background()
	res, err := rpc.TransactionStatusWithReceipts(ctx, txnHash, account)
	if err != nil {
		panic(err)
	}

	spew.Dump(res)
}
```
